stable/stolon/values.yaml

```diff
@@ -0,0 +1,86 @@
+image:
+  repository: sorintlab/stolon
+  tag: v0.10.0-pg9.6
+  pullPolicy: Always
+
+debug: false
+
+persistence:
+  enabled: true
+  storageClassName: default
+  size: 10Gi
+  accessModes:
+    - ReadWriteOnce
+
+rbac:
+  create: true
+
+serviceAccount:
+  create: true
+  # The name of the ServiceAccount to use.
+  # If not set and create is true, a name is generated using the fullname template
+  # name: my-name
+
+superuserUsername: "stolon"
+replicationUsername: "repluser"
+
+## password for the replication user (default is 40 random chars)
+# replicationPassword: "replPassword"
+
+## password for the superuser (default is 40 random chars)
+# superuserPassword: ""
+
+store:
+  backend: "consul"
```

Comment:
This differs from the value set in the README. I'm also not sure if either consul or etcd will get this through CI, as the Chart does not deploy them nor does it set CI overrides to install them. This might be an issue once this is ok-to-test.

Reply:
Updated to include consul requirement.